src/components/App.js

```diff
@@ -141,6 +150,40 @@ class App extends Component<Props, State> {
     shortcuts.off(L10N.getStr("gotoLineModal.key2"), this.toggleQuickOpenModal);
 
     shortcuts.off("Escape", this.onEscape);
+
+    shortcuts.off("Command+Left", this.goToPreviousTab);
+    shortcuts.off("Command+Right", this.goToNextTab);
+  }
+
+  goToNextOrPreviousTab(e, offset) {
+    e.preventDefault();
+
+    const { selectedSource, tabSources, selectSpecificSource } = this.props;
+
+    // There needs to be multiple sources for any action to be taken
+    if (!selectedSource || this.props.tabSources.size < 2) {
+      return;
+    }
+
+    // Get the index of the source to be selected
+    const currentIndex = tabSources.indexOf(this.props.selectedSource);
+    if (currentIndex < 0) {
+      return;
+    }
+
+    // Calculate the next index
+    let nextIndex = 0;
+
+    // Next
+    if (offset) {
```

Comment:
In my opinion, `offset` can be a bit confusing.
Can we separate this into two functions? that would make it more readable.